src/main/java/task/Task.java

```diff
@@ -0,0 +1,66 @@
+package task;
+
+/**
+ *
+ */
+
+public class Task {
```

Comment:
@uberSaiyan says task should be abstract in if you are not going to instantiate it.